docs/pages/includes/permission-warning.mdx

```diff
@@ -8,5 +8,5 @@
   3. We encourage adherence to the *Principle of Least Privilege* (PoLP) and *Zero Admin* best practices. Don't give users the `admin` role when giving them the more restrictive `access,editor` roles will do instead.
   4. Saving tokens into a file rather than sharing tokens directly *as strings*.
 
-  Learn more about [Teleport Role-Based Access Control](../access-controls/introduction.mdx) best practices.
-</Admonition>
\ No newline at end of file
+  Learn more about [Teleport Role-Based Access Control](https://goteleport.com/docs/access-controls/introduction/) best practices.
```

Comment:
the relative URL should work OK,  has a benefit of validation, why it's not working here?

Reply:
ah, I see that's an include and relative links do not work in includes

Reply:
Totally my fault:

1. I tested this using `markdown-lint` but not `markdown-lint-external-links` since it was not an external link. (Good news is that `markdown-lint-external-links` checks for all `https` links). The former only checks by syntax not resolved path.
2. The relative link is relative to the file the include is placed in not the include itself.

🤦